### PR TITLE
Agomanio patch 1

### PR DESCRIPTION
**Pull request by Infinitienet**
add agoman token info and logo in trust wallet.

### CODE REVIEW

**Comment by agomanio** [comment]
Hi my dear bot
I send the TWTs and this is the transaction id: 66E729B4626E7AD46BD29945A3A14119E8C07DF748DEFC7A383881DE2770E4F9
please merge it.

File: blockchains/smartchain/assets/0x95E71FD2e8a46D3dA0379bb79a2605E843BFB55a/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Agoman",
+    "symbol": "AGO",
+    "type": "BEP20",
+    "decimals": 18,
+    "description": "A solution for Authentication of users and businesses on blockchain networks",
+    "website": "https://agoman.io",
+    "explorer": "https://bscscan.com/token/0x95e71fd2e8a46d3da0379bb79a2605e843bfb55a",
+    "status": "active",
+    "id": "0x95e71fd2e8a46d3da0379bb79a2605e843bfb55a"
```
Comment (vikmeup) [inline blockchains/smartchain/assets/0x95E71FD2e8a46D3dA0379bb79a2605E843BFB55a/info.json:10]:
```suggestion
    "id": "0x95E71FD2e8a46D3dA0379bb79a2605E843BFB55a"
```

**Comment by catenocrypt** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 2500 holder addresses and 5000 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863).
This token has no real usage yet (5 holders). Closing PR for now. Please come back to us when all criteria are met.  Multi-chain and staking tokens may get special treatment, let us know if this is the case. 

https://bscscan.com/token/0x95E71FD2e8a46D3dA0379bb79a2605E843BFB55a

**Comment by agomanio** [comment]
we should pay 250twt again for next request?

**Comment by agomanio** [comment]
> As per our policy, brand new tokens are not included, there is a minimum requirement of 2500 holder addresses and 5000 transactions, in order to filter out scam tokens.
> See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863).
> This token has no real usage yet (5 holders). Closing PR for now. Please come back to us when all criteria are met. Multi-chain and staking tokens may get special treatment, let us know if this is the case.
> 
> https://bscscan.com/token/0x95E71FD2e8a46D3dA0379bb79a2605E843BFB55a

we should pay 250twt again for next request?

**Comment by catenocrypt** [comment]
No, this fee can be reconsiderd (no matter if this PR reopened, or new PR created)

**Comment by agomanio** [comment]
Can I get my payment back?

**Comment by catenocrypt** [comment]
Sorry, no refund.

**Comment by agomanio** [comment]
ok thank you
can we use again that payment for reveiew again?

**Comment by catenocrypt** [comment]
Yes, you can request to reopen this PR later if conditions change (or create a new PR if technically simpler)

**Pull request closed without merging**